committee-steering/governance/README.md

```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the steering committee to gain approval.
+  This can simply be submitting a PR and sending mail to [steering@kubernetes.io].
```

Comment:
Nit: the Markdown spec defines an [implicit link name shortcut][1] (e.g. `[steering@kubernetes.io][]`), but leaving the name brackets off entirely seems to be an extention.  Are we targetting a specific Markdown flavor here?

[1]: https://daringfireball.net/projects/markdown/syntax#link

Reply:
@wking links are at the bottom of the doc (this line is on line 62)

Reply:
> links are at the bottom of the doc (this line is on line 62)

Right, and those are fine.  But the syntax used in *this* line (withought the trailng `[]`) isn't in the spec.  It's the "completely omit" case from the [Kramdown spec][1], but I don't know how portable it is to other Markdown engines.

[1]: https://kramdown.gettalong.org/syntax.html#reference-links

Reply:
I think this is fine.  We do it elsewhere in the repo.

Reply:
This is already used in this file and I was just following the pattern.  If and when we start using the content with a markdown parser that doesn't respect this syntax we can look to fix it. As it is, this is part of the commonmark spec which github users (https://spec.commonmark.org/0.28/#shortcut-reference-link).  But the state of "spec" in the markdown world is a disaster and I really don't want to have that argument.